pulsar-broker-auth-sasl/src/test/java/org/apache/pulsar/broker/authentication/ProxySaslAuthenticationTest.java

```diff
@@ -182,6 +183,10 @@ protected void setup() throws Exception {
 		conf.setAuthenticationEnabled(true);
 		conf.setSaslJaasClientAllowedIds(".*" + localHostname + ".*");
 		conf.setSaslJaasServerSectionName("PulsarBroker");
+		File secretKeyFile = File.createTempFile("saslRoleTokenSignerSecret", ".key");
+		secretKeyFile.deleteOnExit();
```

Comment:
please do not use deleteOnExit but delete this file in the After/Teardown method or in a finally block